### PR TITLE
Make `kBaselineTitle` and `kComparisonTitle` global constants

### PR DESCRIPTION
**Pull request by akopich**
Tests: Unit, Manual
Bug: http://b/239166535

### CODE REVIEW

File: src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h [hunk 1]
```diff
@@ -172,26 +169,24 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
...
-                        *function_name, slowdown_percent, comparison_title->toStdString(),
-                        baseline_title->toStdString()));
+  [[nodiscard]] static QString MakeTooltipForSlowdownColumn(const std::string* function_name,
+                                                            double slowdown_percent) {
+    return QString::fromStdString(absl::StrFormat(
+        "The function \"%s\" is  %.3f%%\n"
+        "slower in %s capture that it was in %s capture.\n"
+        "Negative percentage represent a speed-up.",
+        *function_name, slowdown_percent, orbit_mizar_base::kBaselineTitle->toStdString(),
+        orbit_mizar_base::kComparisonTitle->toStdString()));
```
Comment (beckerhe) [inline src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h:179]: Nit: YOu could consider using QString::arg instead of absl::StrFormat. This would avoid conversions to std::string. (If you prefer the current way, this is also fine.)

File: src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h [hunk 2]
```diff
@@ -172,26 +169,24 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
...
-                        "Negative percentage represent a speed-up.",
-                        *function_name, percent_of_slowdown, comparison_title->toStdString(),
-                        baseline_title->toStdString()));
+      const std::string* function_name, double percent_of_slowdown) {
+    return QString::fromStdString(absl::StrFormat(
+        "The slowdown of function \"%s\" constitutes  %.3f%%\n"
+        "of the total frametime slowdown in %s capture compared to %s capture.\n"
+        "Negative percentage represent a speed-up.",
+        *function_name, percent_of_slowdown, orbit_mizar_base::kBaselineTitle->toStdString(),
+        orbit_mizar_base::kComparisonTitle->toStdString()));
```
Comment (beckerhe) [inline src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h:189]: Same as above. But also a Nit.

File: src/MizarBase/include/MizarBase/Titles.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#include <QString>
+
+#include "MizarBase/BaselineOrComparison.h"
+
+namespace orbit_mizar_base {
+
+static const orbit_mizar_base::Baseline<QString> kBaselineTitle =
+    orbit_mizar_base::Baseline<QString>(QStringLiteral("Baseline"));
+static const orbit_mizar_base::Comparison<QString> kComparisonTitle =
+    orbit_mizar_base::Comparison<QString>(QStringLiteral("Comparison"));
```
Comment (beckerhe) [inline src/MizarBase/include/MizarBase/Titles.h:17]: This is fine for your current usage, but depending on where you want to use the constants in the future you could quickly end up in static initialization order fiasco (I think - I'm not entirely sure whether defining those in a header could save you from that. If you don't know for sure I wouldn't count on it.).

I'd recommend to use some type that can be constant initialized - like `std::string_view` and then create QStrings on-the-fly when you need them. This might have more overhead due to QStrings being constructed multiple times, but I assume you're not planning on constructing such QStrings in a hot loop.

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/MizarBase/include/MizarBase/Titles.h (hunk 3 above)
Comment (akopich) [reply src/MizarBase/include/MizarBase/Titles.h:17]: TIL static initialization order fiasco. Thanks. 

Let's replace the two constants with two free functions returning a ref to a local static object.

File: src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h (hunk 1 above)
Comment (akopich) [reply src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h:179]: Yeah, good point. But wouldn't this reduce consistency, as the rest of the class relies on `absl::StrFormat`? 

Anyway, this is orthogonal to the PR.

Comment (beckerhe) [reply src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h]:
```
But wouldn't this reduce consistency, as the rest of the class relies on absl::StrFormat
```

Yes, in a way. That's why I left the choice to you.

Nevertheless there are situations where QString can't be avoided because it has fully fledged encoding (UTF-8) support while std::string is merely a bunch of bytes. absl::StrFormat also can't handle anything that goes beyond ASCII.

TLDR: If we wanted to support other languages in the future, then we needed to move all formatting to QString::arg anyway. Hence there will always be use cases for both in the code base.

But you're right. It's orthogonal.
